scripts/kind-e2e/e2e.sh

```diff
@@ -410,18 +442,17 @@ elif [[ $5 = helm ]]; then
     helm=true
     setup_cluster2_gateway
     setup_cluster3_gateway
-    for i in 2 3; do
-      context=cluster$i
-      kubectl config use-context $context
-
-      create_subm_vars
-      verify_subm_engine_pod
-      verify_subm_routeagent_pod
-      verify_subm_engine_container
-      verify_subm_routeagent_container
-      verify_subm_engine_secrets
-      verify_subm_routeagent_secrets
-    done
+#    for i in 2 3; do
```

Comment:
why commenting them all? can we run them when running the operator? may be we don't really need tbh... as long as the E2E tests run.

Reply:
The goal is to make sure the conditions they assert apply to both the Helm and Operator-based deployments, thereby verifying they are substantially equal. We at least need an option to run them against both imho, like with a flag that's disabled by default.